octoprint_mrbeam/templates/settings/lasercutterprofiles_settings.jinja2

```diff
@@ -66,21 +66,23 @@
             <div class="control-group">
                 <label class="control-label">{{ _('Working Area Size') }}</label>
                 <div class="controls form-inline">
-                    <label>{{ _('X') }}</label>
+                    <label>X</label>
                     <div class="input-append">
                         <input type="number" step="1" class="input-mini text-right" data-bind="value: editorVolumeWidth">
-                        <span class="add-on">mm</span>
+                        {# Translators: millimeters, unit of length #}
+                        <span class="add-on">{{ _('mm') }}</span>
                     </div>
                 </div>
                 <div class="controls form-inline">
-                    <label>{{ _('Y') }}</label>
+                    <label>Y</label>
                     <div class="input-append">
                         <input type="number" step="1" class="input-mini text-right" data-bind="value: editorVolumeDepth">
-                        <span class="add-on">mm</span>
+                        {# Translators: millimeters, unit of length #}
```

Comment:
no need for translations in this file; it's for internal use only.

Reply:
removed all translations from this file